data/json/monsters/insect_spider.json

```
@@ -886,7 +886,7 @@
     "dissect": "dissect_insect_sample_single",
     "anger_triggers": [ "PLAYER_WEAK", "STALK" ],
     "fear_triggers": [ "HURT", "FIRE" ],
-    "upgrades": { "age_grow": 40, "into": "mon_dragonfly_mega" },
+    "upgrades": { "half_life": 40, "into": "mon_dragonfly_mega" },
```

```suggestion
    "upgrades": { "age_grow": 60, "into": "mon_dragonfly_mega" },
```
Bumping up the reproductive form's latency was a thing I wanted to do anyway, though.